scavenger-frontend/src/components/snapshot/SnapshotDetail.vue

```diff
@@ -328,6 +328,17 @@ export default {
     resizeHorizontal(size) {
       localStorage.setItem("scavenger.snapshot.horizontal-size", size);
     },
+    isStartsWithSignature(querySignature, signature) {
+      if (!querySignature.startsWith(signature)) {
+        return false;
+      }
+
+      if (querySignature === signature) {
+        return true;
+      }
+
+      return querySignature.substring(signature.length).startsWith(".");
```

Comment:
What is this line means?

Reply:
The package must be followed by a class, so check for a `.` as well.